src/index.tsx

```diff
@@ -82,7 +82,7 @@ const experimentalFeatureWarnings = ({ steps }: NormalisedSdkOptions) => {
 const isSMSCountryCodeValid = (smsNumberCountryCode: string) => {
   // If you need to refactor this code, remember not to introduce large libraries such as
   // libphonenumber-js in the main bundle!
-  const countries = getCountryCodes(labels)
+  const countries = getCountries(metadata) as string[] // react-phone-number-input returns an array of CountryCode type contrary to documentation
```

Comment:
Is this the only way of doing this validation? I don't like the idea of importing this large library globally for this validation only tbh...

Reply:
Agreed about having to introduce the large library just for the country code validation. I've had a think & what I can try is hacky, but we could make use of the `labels` value to do the validation like it used to. The file it was originally imported from still exists, just renamed/restructured.